client/src/style/About.css

```diff
@@ -15,25 +15,35 @@
   color: #ffffff;
   padding-left: 5.5px;
 }
-.center {
+.centerAbout {
```

Comment:
Why the change in the naming scheme? This adds complexity since it's just a center and it must be in the about section since it is named About.css.

This is more of a stylistic choice but I am just trying to understand your objective here

Reply:
The About.css was somehow impacting and creating a bug in courseselection so I had to change it around.

Reply:
And when CourseSelection has a center attribute, does this issue still occur?

Reply:
no, but might as well make the diiv names better to understand

Reply:
But isn't center clear? It's more complex by adding the about since that implies there is another center here.

Reply:
ahh ok ill change it back to center then.

Reply:
@bayoumi17m Changed. Please review again.